lib/active_model/serializer.rb

```diff
@@ -338,7 +338,7 @@ def to_json(*args)
     # object including the root.
     def as_json(options={})
       options ||= {}
-      if root = options.fetch(:root, @options.fetch(:root, root_name))
+      if root = options.to_hash.fetch(:root, @options.fetch(:root, root_name))
```

Comment:
For reference: https://travis-ci.org/rails-api/active_model_serializers/builds/43605004